python/sglang/srt/server.py

```diff
@@ -429,14 +435,6 @@ def launch_server(
 
     try:
         # Listen for HTTP requests
-        LOGGING_CONFIG["formatters"]["default"][
```

Comment:
QQ Why did you remove the logging format code?

Reply:
Sorry, missed this when pulling in the latest changes!